extra/3-magic-8-ball.js

```diff
@@ -60,8 +101,21 @@ function shakeBall() {
 */
 function checkAnswer(answer) {
   //Write your code in here
+
+  //console.log(answer);
+  if (answer=="It is certain."||answer=="It is decidedly so."||answer== "Without a doubt."||answer=="Yes - definitely."||answer=="You may rely on it."){
+    return "very positive";
+  } else if (answer == "As I see it, yes."||answer == "Most likely."||answer == "Outlook good."||answer ==  "Yes."||answer ==  "Signs point to yes."){
+    return "positive";
+  }else if (answer == "Reply hazy, try again."||answer == "Ask again later."||answer == "Better not tell you now."||answer == "Cannot predict now."||answer == "Concentrate and ask again."){
+    return "negative";
+  }else if (answer == "Don't count on it."||answer ==  "My reply is no."||answer ==  "My sources say no."||answer ==  "Outlook not so good."||answer ==  "Very doubtful."){
+    return "very negative";
+  }
```

Comment:
This works well - you could also put all the possible answers in 1 array, find the index of the answer and use that to categorise it. This is a tricky exercise though, and there is no elegant answer. When you learn about objects later in the course, you might see better ways of doing it.

Reply:
thank you Karen